src/devices/src/virtio/mod.rs

```diff
@@ -6,27 +6,27 @@
 // found in the THIRD-PARTY file.
 
 //! Implements virtio devices, queues, and transport mechanisms.
+
 use std::any::Any;
-use std::io::Error as IOError;
 
 pub mod balloon;
 pub mod block;
-pub mod device;
+mod device;
 mod mmio;
 pub mod net;
 pub mod persist;
 mod queue;
 pub mod test_utils;
 pub mod vsock;
 
-pub use self::balloon::*;
-pub use self::block::*;
-pub use self::device::*;
-pub use self::mmio::*;
-pub use self::net::*;
-pub use self::persist::*;
-pub use self::queue::*;
-pub use self::vsock::*;
+pub use self::balloon::Balloon;
+pub use self::block::Block;
+pub use self::device::{IrqType, VirtioDevice};
+pub use self::mmio::MmioTransport;
+pub use self::net::Net;
+pub use self::persist::QueueState;
+pub use self::queue::Queue;
+pub use self::vsock::{Vsock, VsockUnixBackend, TYPE_VSOCK};
```

Comment:
Nit: This can be simplified.
```suggestion
pub use balloon::Balloon;
pub use block::Block;
pub use device::{IrqType, VirtioDevice};
pub use mmio::MmioTransport;
pub use net::Net;
pub use persist::QueueState;
pub use queue::Queue;
pub use vsock::{Vsock, VsockUnixBackend, TYPE_VSOCK};
```